src/status_im/contexts/chat/messenger/composer/actions/view.cljs

```diff
@@ -30,7 +30,11 @@
         contact-customization-color (when (= chat-type constants/one-to-one-chat-type)
                                       (rf/sub [:contacts/contact-customization-color-by-address
                                                chat-id]))
-        on-press                    (rn/use-callback #(send-message input-ref edit btn-opacity) [edit])]
+        on-press                    (rn/use-callback
+                                     (fn []
+                                       (js/requestAnimationFrame
+                                        #(send-message input-ref edit btn-opacity)))
```

Comment:
Thank you @seanstrom for fixing this issue. Please can you add a comment to explain how this fixes the issue.